content/community/conferences.md

```diff
@@ -44,6 +44,11 @@ May 17-18 in Paris, France
 
 [Website](https://www.react-europe.org) - [Twitter](https://twitter.com/ReactEurope) - [Facebook](https://www.facebook.com/ReactEurope)
 
+### ReactNext 2018
+September 6 in Tel Aviv, Israel
+
+[Website](https://react-next.com) - [Twitter](https://twitter.com/ReactNext) - [Facebook](https://facebook.com/ReactNext2016)
```

Comment:
Just to quickly confirm: The Facebook link for React Next 2018 is facebook.com/ReactNext2016?